src/docs/resources/platform-adaptations.md

```diff
@@ -0,0 +1,218 @@
+
+<div class="container">
+  <div class="row">
+    <div class="col-sm text-center">
+      <figure class="figure">
+        <img src="../../images/platform-adaptations/navigation-android.gif" class="figure-img img-fluid rounded" alt="An animation of the page transition on Android pre-Android P" />
+        <figcaption class="figure-caption">
+          Android Pre-P
+        </figcaption>
+      </figure>
+    </div>
+    <div class="col-sm">
+      <figure class="figure text-center">
+        <img src="../../images/platform-adaptations/navigation-android-p.gif" class="figure-img img-fluid rounded" alt="An animation of the page transition on Android on Android P" />
+        <figcaption class="figure-caption">
+          Android Post-P
+        </figcaption>
+      </figure>
+    </div>
+    <div class="col-sm">
+      <figure class="figure text-center">
+        <img src="../../images/platform-adaptations/navigation-ios-nav-bar.gif" class="figure-img img-fluid rounded" alt="An animation of the nav bar transitions during a page transition on iOS" />
+        <figcaption class="figure-caption">
+          iOS Nav Bar
+        </figcaption>
+      </figure>
+    </div>
+  </div>
+</div>
+
+### Back Navigation
+
+On **Android**, the OS back button, by default, is sent to Flutter and pops the top
+route of the [WidgetsApp](https://docs.flutter.io/flutter/widgets/WidgetsApp-class.html)'s
+Navigator.
+
+On **iOS**, an edge swipe gesture can be used to pop the top route.
+
+## Scrolling
+
+### Physics Simulation
+
+### Overscroll Behavior
+
+On **Android**, scrolling past the edge of a scrollable shows an [overscroll 
+glow indicator](https://docs.flutter.io/flutter/widgets/GlowingOverscrollIndicator-class.html)
+(based on the color of the current Material theme).
+
+On **iOS**, scrolling past the edge of a scrollable [overscrolls](https://docs.flutter.io/flutter/widgets/BouncingScrollPhysics-class.html)
+with increasing resistance and snaps back. 
+
+## Typography
+
+When using the Material package, the typography automatically default to the
+font family appropriate for the platform. On Android, the Roboto font is used.
+On iOS, the OS's San Francisco font family is used.
+
+When using the Cupertino package, the [default theme](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/cupertino/text_theme.dart)
+always uses the San Francisco font.
+
+The San Francisco font license limits its usage to software running on iOS,
+macOS or tvOS only. Therefore a fallback font is used when running on Android
+if the platform is debug-overridden to iOS or the default Cupertino theme is
+used.
+
+## Iconography
+
+When using the Material package, certain icons automatically show different
+graphics depending on the platform. For instance, the overflow button's 3 dots
+are vertical on iOS and horizontal on Android. The back button is a simple
+chevron on iOS and has a stem/shaft on Android.
+
+## Haptic Feedback
```

Comment:
=> Haptic feedback

I know what haptic feedback is, because I recently learned it when my daughter was going through college, but will most ppl?

Reply:
I think so. It's been in the first smartphones.